sdk/program/Cargo.toml

```diff
@@ -34,6 +34,8 @@ sha3 = { workspace = true }
 solana-atomic-u64 = { workspace = true }
 solana-frozen-abi = { workspace = true, optional = true }
 solana-frozen-abi-macro = { workspace = true, optional = true }
+solana-hash = { workspace = true, features = ["borsh", "bytemuck", "serde"] }
```

Comment:
I think we should sneak this crate's `borsh` dependency behind the program SDK's `borsh` feature so nobody is forced to bring in `borsh`.

Reply:
good spot